packages/stark-ui/src/modules/app-data/components/app-data.component.html

```diff
@@ -9,25 +9,40 @@
 		<ng-content select=".summary-slot"></ng-content>
 	</div>
 
-	<button mat-icon-button aria-label="Application Data" (click)="toggleDetail()" [ngClass]="{ 'is-open': !isDetailHidden }" mat-button>
+	<button
+		mat-icon-button
+		aria-label="Application Data"
+		#dropdownDetailTrigger="matMenuTrigger"
+		[matMenuTriggerFor]="dropdownDetail"
+		[class.is-open]="dropdownDetailTrigger.menuOpen"
+		mat-button
+	>
 		<mat-icon svgIcon="menu-down"></mat-icon>
 	</button>
-	<div class="stark-app-data-detail stark-fade-animation" *ngIf="!isDetailHidden">
-		<ng-container *ngTemplateOutlet="appDataDetail"></ng-container>
-	</div>
+	<mat-menu #dropdownDetail="matMenu" xPosition="before" yPosition="below" class="stark-app-data dropdown-detail">
+		<ng-template matMenuContent>
```

Comment:
Question: instead of repeating this `ng-template` block twice, could we just add it to the `<ng-template #appDataDetail>` that we have at the top of this file?

Or at least move there as much code as possible to avoid duplication and potential mistakes 😉

Reply:
Good idea! Actually, I tried already the following code

```html
<ng-template #appDataDetail>
  <div class="stark-app-data-detail" (click)="$event.stopPropagation()">
    <ng-content select=".detail-slot"></ng-content>
  </div>
</ng-template>

<!-- ... -->
<mat-menu #dropdownDetail="matMenu" xPosition="before" yPosition="below" class="stark-app-data dropdown-detail">
  <ng-container matMenuContent *ngTemplateOutlet="appDataDetail"></ng-container>
</mat-menu>
```

But it was not woking well (I had the error message again).
If I only move the line `<div class="stark-app-data-detail" (click)="$event.stopPropagation()">`, it's obviously working well 😊 
Thanks for your remark!